corehq/apps/app_manager/tests/data/suite/schedule-entry.xml

```diff
@@ -6,11 +6,20 @@
             </text>
         </title>
         <variables>
-            <next_a1e369 function="if(current_schedule_phase = '', date(edd) + int(instance('schedule:m1:f0')/schedule/visit[@late_window = '' or today() &lt;= (date(edd) + int(@due) + int(@late_window))][1]/@due), if(current_schedule_phase = 1 and edd != '' and (instance('schedule:m1:f0')/schedule/@expires = '' or today() &lt; (date(edd) + instance('schedule:m1:f0')/schedule/@expires)), date(edd) + int(instance('schedule:m1:f0')/schedule/visit[@id &gt; last_visit_number_a1e369][@late_window = '' or today() &lt;= (date(edd) + int(@due) + int(@late_window))][1]/@due), 0))"/>
-            <next_a983e9 function="if(current_schedule_phase = 2 and dob != '' and (instance('schedule:m1:f1')/schedule/@expires = '' or today() &lt; (date(dob) + instance('schedule:m1:f1')/schedule/@expires)), date(dob) + int(instance('schedule:m1:f1')/schedule/visit[@id &gt; last_visit_number_a983e9][@late_window = '' or today() &lt;= (date(dob) + int(@due) + int(@late_window))][1]/@due), 0)"/>
-            <next_39f0ea function="if(current_schedule_phase = 3 and dob != '' and (instance('schedule:m1:f2')/schedule/@expires = '' or today() &lt; (date(dob) + instance('schedule:m1:f2')/schedule/@expires)), date(dob) + int(instance('schedule:m1:f2')/schedule/visit[@id &gt; last_visit_number_39f0ea][@late_window = '' or today() &lt;= (date(dob) + int(@due) + int(@late_window))][1]/@due), 0)"/>
-            <next_due function="min(next_a1e369,next_a983e9,next_39f0ea)"/>
-            <is_late function="next_due &lt; today()"/>
+            <anchor_हिन्दी function="edd"/>
```

Comment:
but yes, apparently that is well formed XML. You can use this swanky JS feature I just discovered: `DOMParser()`
```js
var parser = new DOMParser()
var parsed = parser.parseFromString("<anchor_हिन्दी function='edd'/>", "application/xml")
parsed.activeElement.tagName === "parseerror" ? "error" : "good!"
```